driver-sync/src/test/functional/com/mongodb/client/AbstractClientSideEncryptionTest.java

```diff
@@ -195,10 +196,7 @@ public void setUp() {
         }
 
         Map<String, Object> extraOptions = new HashMap<>();
-        String cryptSharedLibPath = System.getProperty("org.mongodb.test.crypt.shared.lib.path", "");
-        if (!cryptSharedLibPath.isEmpty()) {
-            extraOptions.put("cryptSharedLibPath", cryptSharedLibPath);
-        }
```

Comment:
I also needed to use this Java system property, so I refactored this code a bit.